wiremock/mappings/ProbationOffenderSearchApi_SearchByNomsNumber.json

```diff
@@ -0,0 +1,156 @@
+{
+  "id": "05c36ca3-f66c-4ffe-917e-65aff9567af2",
+  "request": {
+    "method": "POST",
+    "urlPathPattern": "/search.*",
+    "bodyPatterns": [
+      {
+          "equalToJson": { "nomsNumber": "A1234AI" }
+      }
+  ]
+  },
+  "response": {
+    "headers": {
+      "Content-Type": "application/json"
+    },
+    "status": 200,
+    "jsonBody": [
+      {
+        "offenderId": 1,
+        "softDeleted": false,
+        "otherIds": {
+          "crn": "X320741",
+          "pncNumber": "2018/0123456X",
+          "croNumber": "SF80/655108T",
+          "nomsNumber": "A1234AI",
+          "previousCrn": "X10001"
+        },
+        "title": "Mr",
+        "firstName": "Aadland",
+        "middleNames": [],
+        "surname": "Bertrand",
+        "previousSurnames": [],
+        "dateOfBirth": "1978-01-06",
+        "gender": "Male",
+        "currentDisposal": "1",
+        "currentExclusion": false,
+        "currentRestriction": false,
+        "partitionArea": "some name",
+        "offenderAliases": [
+        ],
+        "contactDetails": {
+          "addresses": [
+            {
+              "from": "2018-01-16",
+              "addressNumber": "12",
+              "buildingName": "string",
+              "county": "string",
+              "district": "string",
+              "noFixedAbode": true,
+              "notes": "string",
+              "postcode": "NE2 2SW",
+              "streetName": "Church Street Lane",
+              "telephoneNumber": "string",
+              "town": "Newcastle under Lyme",
+              "status": {
+                "code":"m",
+                "description": "main"
+              },
+              "createdDateTime": "2022-08-12T10:30:27",
+              "lastUpdatedDateTime": "2022-08-12T10:30:27"
+            }
+          ],
+          "emailAddresses": [],
+          "phoneNumbers": [],
+          "allowSMS": true
+        },
+        "offenderProfile": {
+          "riskColour": "red",
+          "ethnicity": "White British",
+          "immigrationStatus": null,
+          "nationality": "British",
+          "notes": "string",
+          "offenderDetails": "string",
```

Comment:
`offenderDetails` might be something like "Induction pack completed on IA on 12/07/2022. SD Updated phone number 07000 000 000"